opsdroid/configuration/schema.yaml

```diff
@@ -0,0 +1,118 @@
+welcome-message: bool()
+
+connectors:
+  list(include('websocket', required=False),
+  include('shell', required=False),
+  include('ciscospark',required=False),
+  include('facebook',required=False),
+  include('github',required=False),
+  include('matrix',required=False),
+  include('slack',required=False),
+  include('telegram',required=False),
+  include('twitter',required=False))
+
+databases:
+  list(include('mongo', required=False),
+  include('redis', required=False),
+  include('sqlite', required=False),required=False)
+
+skills: list(include('skill',required=False))
+
+parsers: list(include('parser',required=False),required=False)
```

Comment:
Parsers is optional.

Reply:
required = False: this means optional.
 Added config (valid_without_db_and_parsers.yaml) and test verifying this (test_load_config_valid_without_db_and_parsers)